app/views/placements/wizards/add_mentor_wizard/_check_your_answers_step.html.erb

```diff
@@ -1,11 +1,12 @@
+<% caption = current_user.support_user? ? "#{t(".add_mentor")} #{contextual_text}" : t(".add_mentor") %>
 <% content_for :page_title, t(".page_title", contextual_text:) %>
 
 <div class="govuk-width-container">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <%= form_with(model: check_your_answers_step, url: current_step_path, method: :put) do |f| %>
         <label class="govuk-label govuk-label--l">
-          <span class="govuk-caption-l"><%= t(".add_mentor_caption", contextual_text:) %></span>
```

Comment:
Does this mean we can also remove the `.add_mentor_caption` translation?

https://github.com/DFE-Digital/itt-mentor-services/blob/8dd9c6f836570ddd913ee73c5fc2af3e92ebc29a/config/locales/en/placements/wizards/add_mentor_wizard.yml#L23

Looks like this might also be a dead translation, so could be worth removing while we're at it:

https://github.com/DFE-Digital/itt-mentor-services/blob/60b5b8468f05e2f590ab42c99c197ca47608a90e/config/locales/en/placements/wizards/add_mentor_wizard.yml#L8